Configuration/StandardSequences/python/Eras.py

```diff
@@ -44,8 +44,8 @@ def __init__(self):
                            'phase2_hgcal', 'phase2_muon', 'phase2_timing',
                            'phase2_timing_layer','phase2_hcal',
                            'trackingLowPU', 'trackingPhase1', 'ctpps_2016', 'trackingPhase2PU140',
-                           'tracker_apv_vfp30_2016', 'run2_miniAOD_80XLegacy','run2_miniAOD_94XFall17', 'run2_nanoAOD_92X',
-                           'hcalHardcodeConditions', 'hcalSkipPacker']
+                           'tracker_apv_vfp30_2016', 'run2_miniAOD_80XLegacy', 'run2_nanoAOD_92X',
+                           'hcalHardcodeConditions', 'hcalSkipPacker', 'run2_miniAOD_94XFall17']
```

Comment:
just in case somebody else decides to edit this file while we integrate, please remove the edits which are not required.
It may be easier to completely remove/remake the commit that introduced this file in the first place.

Reply:
also, keeping the ```run2_miniAOD_*``` grouped here (as in the master version now) would reduce the randomness of where to put the next one

Reply:
@roger-wolf 
please let me know if you will be able to clean up changes in this file
or if you are perhaps waiting for some other inputs.
Thank you.

Reply:
Hi,
I did not understand that this requires action. I'm moving the run2_miniAOD_94XFall17 era now in the place where I had originally put it. I don't know how to completely revert a commit. I hope the results are equally fine.